ios/PurchasesHybridCommon/PurchasesHybridCommonTests/StoreProduct+HybridAdditionsTests.swift

```diff
@@ -0,0 +1,178 @@
+//
+//  StoreProduct+HybridAdditionsTests.swift
+//  PurchasesHybridCommonTests
+//
+//  Created by Cesar de la Vega on 12/27/22.
+//  Copyright © 2022 RevenueCat. All rights reserved.
+//
+
+import Quick
+import Nimble
+import StoreKit
+@testable import RevenueCat
+
+class StoreProductHybridAdditionsTests: QuickSpec {
+
+    private func storeProductDictionaryWith(locale: Locale = Locale(identifier: "en_US"),
```

Comment:
The Swifty thing would be 
```suggestion
    private func storeProductDictionary(with locale: Locale = Locale(identifier: "en_US"),
```